pkg/remote/remote.go

```diff
@@ -20,10 +22,10 @@ func IsSupported(remote string) bool {
 	return false
 }
 
-func Activate(remote string, alerter *alerter.Alerter) error {
+func Activate(remote string, alerter *alerter.Alerter, providerLibrary *terraform.ProviderLibrary, supplierLibrary *resource.SupplierLibrary) error {
 	switch remote {
 	case aws.RemoteAWSTerraform:
-		return aws.Init(alerter)
+		return aws.Init(alerter, providerLibrary, supplierLibrary)
```

Comment:
Shouldn't we pass just the AWS terraform to aws init directly ?

Reply:
We are creating the terraform provider whithin the Init